mailpoet/assets/js/src/automation/integrations/woocommerce-subscriptions/steps/subscription-renewed/index.tsx

```diff
@@ -0,0 +1,36 @@
+import { __ } from '@wordpress/i18n';
+import { StepType } from '../../../../editor/store';
+import { Icon } from './icon';
+import { PremiumModalForStepEdit } from '../../../../../common/premium-modal';
+
+const keywords = [
+  __('woocommerce', 'mailpoet'),
+  __('subscription', 'mailpoet'),
+  __('renewed', 'mailpoet'),
+];
+export const step: StepType = {
+  key: 'woocommerce-subscriptions:subscription-renewed',
+  group: 'triggers',
+  title: () => __('Woo Subscription renewed', 'mailpoet'),
+  description: () =>
+    __('Start the automation when a subscription renewed.', 'mailpoet'),
```

Comment:
I would prefer `when a user renews their subscription`, or `when a subscription renews` or `when a subscription is renewed`

Reply:
https://github.com/mailpoet/mailpoet/pull/5264/commits/d24111eda686d0e5643dd27b42db2c359aac3849